src/config/database.ts

```
@@ -28,7 +28,42 @@ if (config.prismaAccelerateUrl && !ACCELERATE_PROTOCOL_RE.test(config.prismaAcce
+// (used for health probes and migrations) so PostgreSQL cancels the statement
+// itself just before Accelerate would.  The direct URL is not used for runtime
+// traffic when Accelerate is active, but this keeps the timeout in sync for
+// anything that bypasses the proxy (e.g. migration runner, health checks).
+//
+// For runtime traffic through Accelerate, keep ACCELERATE_QUERY_TIMEOUT_MS in
+// the Accelerate dashboard ≥ 10 000 ms and ensure your slowest query completes
+// within that window.
+const STATEMENT_TIMEOUT_MS = parseInt(
+  process.env.DB_STATEMENT_TIMEOUT_MS ?? "9000",
+  10,
+);
```

ESLint

[error] Replace `⏎··process.env.DB_STATEMENT_TIMEOUT_MS·??·"9000",⏎··10,⏎` with `process.env.DB_STATEMENT_TIMEOUT_MS·??·"9000",·10`

(prettier/prettier)